### PR TITLE
SA-69188 updated sbt.version to 1.8.0

### PR DESCRIPTION
**Pull request by adrianmatwiejuk**

### CODE REVIEW

**Comment by adrianmatwiejuk** [comment]
com-dotdata dependencies have releases with hashes instead of proper, nice ones, as they are currently PRs as well (https://ramencloud.atlassian.net/browse/SA-69684). Once those get merged I'll replace the versions with proper releases.

**Comment by adrianmatwiejuk** [comment]
TODO release this after approve, use image in bigdata PRs

**Review by iwanek9ld — APPROVED** [review]

**Comment by zak-dd** [comment]
@adrianmatwiejuk Please squash your commits before merging if you don't maintain a clean history on your branch

File: build.sbt [hunk 1]
```diff
@@ -34,13 +40,15 @@ val HadoopDependencies = Seq(
 )
 
 val TestDependencies = Seq(
-  "org.scalatest" %% "scalatest" % ScalaTestVersion % "test->*",
-  "org.scalamock" %% "scalamock-scalatest-support" % ScalamockVersion % "test->*"
+  scalatest,
+  scalacheck,
+  scalamock,
+  scalactic
```
Comment (zak-dd) [inline build.sbt:46]: Those are not test dependencies anymore, as far as I can tell. I look like you made `compile` also depend on them

Comment (iwanek9ld) [reply build.sbt:46]: Good catch. @adrianmatwiejuk Could you take a look?

**Pull request merged**